lib/existing-eks-nginx-observability-construct/rules.yml

```diff
@@ -0,0 +1,31 @@
+groups:
```

Comment:
I like this file. I think you should rename it to nginx-rules... and move it to common folder like other PR.

Reply:
change name and add to common/resources/amp-config